e2e/appActions.js

```diff
@@ -405,19 +406,86 @@ async function selectInspectorTab(page, name) {
     }
 }
 
+/**
+* Waits and asserts that all plot series data on the page
+* is loaded and drawn.
+* @param {import('@playwright/test').Page} page
```

Comment:
I would update this text to add something about why this is css is significant.

I think a lot of folks in the general testing community would find this approach useful and so it would be helpful to explain that this is a unique css property we created, but was relatively simple to implement